src/physics/arcade/World.js

```diff
@@ -1355,6 +1357,9 @@ var World = new Class({
         velocityY = Clamp(velocityY, -maxY, maxY);
 
         body.velocity.set(velocityX, velocityY);
+        if(maxSpeed >= 0 && body.velocity.length() > maxSpeed){
```

Comment:
It looks like existing code stype should:
a) Put a space after the opening if and before the `(`
b) Put the opening curly brace on the next line.

(Look at some of the `if`'s above)

Reply:
Ironically, I personally prefer curly braces on the same line, like this (although I like spaces before them).